### PR TITLE
docs(VNavigationDrawer): add difference between v-model:rail and rail="true" props

### PR DESCRIPTION
**Pull request by gm2211**
## Description
There is a caveat on the docs page for navbar that alerts dev that main contents are do not respond to the resizing of the navbar when `rail` and `expand-on-hover` are set. The warning suggests using `mini-variant.sync` to a data prop, however that:

1. Does not seem to work with the latest version of vuetify / vuejs (was mini-variant replaced by `rail`?)
2. It seems not consistent with 
https://github.com/vuetifyjs/vuetify/blob/master/packages/docs/src/examples/v-navigation-drawer/prop-mini-variant.vue#L6

What seems to work is:

```html
<v-navigation-drawer
     :rail="rail"
     @mouseover="rail = false"
     @mouseleave="rail = true">
</v-navigation-drawer>
```

and

```typescript
<script lang="ts">
export default {
  data() {
    return {
      rail: true
    }
  }
}
</script>
```

### CODE REVIEW

**Comment by johnleider** [comment]
This is done using [v-model:rail](https://play.vuetifyjs.com/#eNqlUstu2zAQ/BWCl7SAKdZNT4ZTpN8R5kBL65qpRBLkSk5h6N+7JCVDcZoHkht3uTOzj7k78V/eV0MPfMO3CJ1vNcJPZRnbDkJ7n585sHowvzUaZ0UT9BFC+WEsaNPOb3j02jaCag5umEvkRPI21SA610C7SZQ3imdmrqxChYy1rs4Y+gDbpPwbmme9Ths79ZDj2lmkFARWtzpGImzEvoVH9tBHNPu/Inpdg9gBHgGs4mcsoRszLELGEhdYZBF1wEWdfFr4CsPplFfIxvFD8LmBtJMX8Fu5mHm+KOXOe0lBOfZWLkxAYayD8TQdYF+8YDrvAjLqGfZsZPvgOnZF/rkq+yaZiGWem1TyBUMPXxNvYSIOvuKFhPR99RCdJe+dElhNH1HxDcuZlCPyFCt+QPRxI2XdWIKRT8wQKgsore/kLZXJ0Fs0HYjGdbfX1XX1TejWH3S1/k77iLj8ryB2YhfcMUIgNsVXC71kgndoTpUkta7WP4rElBOt3sWk8YxbkiDZVAQ6GATy6ztnu4D9d76LmmczpjbIZSOdoDX2T7zYfh3z5u/mbj83fGZLVPdJcVzxpx1TDwXK7/8BDMJwBg==)

File: packages/docs/src/pages/en/components/navigation-drawers.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ The navigation drawer is primarily used to house links to the pages in your appl
 
 <alert type="info">
 
-  The **expand-on-hover** prop does not alter the content area of **v-main**. To have content area respond to **expand-on-hover**, bind **mini-variant.sync** to a data prop.
+  The **expand-on-hover** prop does not alter the content area of **v-main**. To have content area respond to drawer expansion, instead of **expand-on-hover**, you can add create a data prop `rail` and then add this to your nav bar `:rail="rail"` followed by `@mouseover="rail = false"` and `@mouseleave="rail = true"`.
```
Comment (pg-ahenry) [inline packages/docs/src/pages/en/components/navigation-drawers.md:38]:
```suggestion
  The **expand-on-hover** prop does not alter the content area of **v-main**. To have content area respond to **expand-on-hover**, bind **v-model:rail** to a data prop.
```

**Comment by johnleider** [comment]
Can you please update the content?

File: packages/docs/src/pages/en/components/navigation-drawers.md (hunk 1 above)
Comment (MajesticPotatoe) [inline packages/docs/src/pages/en/components/navigation-drawers.md:38]:
```suggestion
  The **expand-on-hover** prop does not alter the content area of **v-main**. To have content area respond to **expand-on-hover**, bind **v-model:rail** to a data prop.
```

**Review by MajesticPotatoe — APPROVED** [review]
👍

**Pull request merged**